src/mono-runtimes/mono-runtimes.projitems

```diff
@@ -159,7 +159,7 @@
 
     <_MonoCrossRuntime Include="cross-arm-win" Condition="$(AndroidSupportedTargetAotAbisForConditionalChecks.Contains (':win-armeabi-v7a:'))">
       <Strip>$(AndroidMxeFullPath)\bin\$(MingwCommandPrefix32)-strip</Strip>
-      <StripFlags>-S</StripFlags>
```

Comment:
Do we not need `-S` because mono SDKs is now stripping outputs?

Reply:
No, `-S` triggers a SIGSEGV for some reasons, and the resulting binary is the same with and without the option.